apps/browser/qml/pages/components/FavoriteGrid.qml

```diff
@@ -113,7 +129,7 @@ IconGridViewBase {
     Component {
         id: iconFetcher
 
-        DataFetcher {
+        IconFetcher {
```

Comment:
Is it really necessary?

Reply:
See rebase, so in next branch

Reply:
I'm also confused by this. After rebasing on top of `next` I had to switch this back to `DataFetcher` to get this to work.